TypeApprovalRegister+forwardings.yaml

```diff
@@ -161,7 +161,7 @@ forwardings:
         uuid: tar-1-0-0-op-s-im-000
     consequent-requests:
       - client-name: NewRelease:://v1/document-approval-status
-        uuid: tar-1-0-0-op-c-im-nr-1-0-0-001
+        uuid: tar-1-0-0-op-c-im-nr-1-0-0-000
```

Comment:
the api-segment of /v1/document-approval-status would be "is" since this service will not be involving in any config changes. so, the uuid would be tar-1-0-0-op-c-is-nr-1-0-0-000

Reply:
Corrected as proposed.